.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# No default owner
+
+# CI-related changes
+Makefile       @status-im/devops @status-im/status-go-guild
+/nix/          @status-im/devops
+/_assets/ci/   @status-im/devops
```

Comment:
You're forgetting:

- `shell.nix`
- `default.nix`
- `/_assets/scripts/`
- `/_assets/hooks/`
- `/_assets/build/`

Reply:
Will add 👍